src/Shopsys/ShopBundle/Model/Administrator/Security/AdministratorUserProvider.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
```

Comment:
missing strict types (travis check even failed on this!)

Reply:
Fixed in 14fcabc9caed7e549ebd245f68c9f5adeb7f1d29